draft-ietf-masque-h3-datagram.md

```diff
@@ -623,68 +306,30 @@ recent draft version from the intersection set. This ensures that both
 endpoints negotiate the same draft version.
 
 
-# The Sec-Use-Datagram-Contexts HTTP Header {#context-hdr}
+# The Capsule-Protocol Header
```

Comment:
Editorial: consider moving this whole section to be part of the definition of Capsule protocol definition section

Reply:
Done